learn/_components/CopyButton.tsx

```diff
@@ -0,0 +1,19 @@
+export function CopyButton(props: { text: string }) {
+  return (
+    <button
+      type="button"
+      aria-label="Copy code to clipboard"
```

Comment:
Same note about buttons and `.sr-only` text here; better to use text than `aria-label` (and to put `aria-hidden` on the SVG, just to be safe).